<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="haile" failures="1" tests="5" name="TestLoaiDeThi" time="0.053" errors="0" timestamp="2021-12-07T14:18:07 ICT">
  <testcase classname="testModel.LoaiDeThiTest" name="getLoaiDe_IDTest_TC01" time="0.005"/>
  <testcase classname="testModel.LoaiDeThiTest" name="getTenLoaiTest_TC02" time="0.001"/>
  <testcase classname="testModel.LoaiDeThiTest" name="setLoaiDe_IDTest_TC03" time="0.001"/>
  <testcase classname="testModel.LoaiDeThiTest" name="setLoaiDe_IDTest_TC04" time="0.001">
    <failure type="org.testng.TestException" message="
Method LoaiDeThiTest.setLoaiDe_IDTest_TC04()[pri:0, instance:testModel.LoaiDeThiTest@247bddad] should have thrown an exception of type class java.lang.Exception">
      <![CDATA[org.testng.TestException: 
Method LoaiDeThiTest.setLoaiDe_IDTest_TC04()[pri:0, instance:testModel.LoaiDeThiTest@247bddad] should have thrown an exception of type class java.lang.Exception
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- setLoaiDe_IDTest_TC04 -->
  <testcase classname="testModel.LoaiDeThiTest" name="setTenLoaiTest" time="0.001"/>
</testsuite> <!-- TestLoaiDeThi -->
